### ISSUE TITLE
Fatal error when php_uname() is disabled for security reasons.

### ISSUE THREAD

**Issue by iamdharmesh** [apimatic/core-lib-php#34]
There are many web hosting providers that disable a number of built-in PHP functions due to security reasons. When the `php_uname()` function is disabled on a web host, this lib throws a fatal error. 

```
PHP Fatal error:  Uncaught Error: Call to undefined function Core\Utils\php_uname() in /Users/**/**/**/**/public/wp-content/plugins/woocommerce-square/vendor/apimatic/core/src/Utils/CoreHelper.php:159
```

Lib should work fine without any fatal error even if `php_uname` is disabled. We may fix this issue by adding the `function_exists()` check on php_uname and using the fallback value here
https://github.com/apimatic/core-lib-php/blob/55cc76f589a62661673ebd5fdf98614d907bf276/src/Utils/CoreHelper.php#L148

Thanks

**Comment by asadali214** [comment]
Hi @iamdharmesh 

Thanks for highlighting this fatal error. We are looking into this, and will circle back soon.

Cheers!

**Comment by asadali214** [comment]
This issue is resolved and the fix is released via version [0.3.1](https://github.com/apimatic/core-lib-php/releases/tag/0.3.1). You might have to update the dependencies using the `composer update` command.

Cheers!

**Comment by iamdharmesh** [comment]
Thanks for the quick fix and release @asadali214. I really appreciate it.
